pints/_plot.py

```diff
@@ -0,0 +1,248 @@
+#
+# Quick diagnostic tools for inference results
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+import pints
+import numpy as np
+from scipy import stats
+#import matplotlib.pyplot as plt
+import pylab as plt
+
+def trace(chain):
+    """
+    Trace plot.
```

Comment:
Please write a better description! Typically a one line paragraph ("This method creates and returns..."). Followed by a second paragraph with any extra description needed, and finally an argument list (like you already started) and a line at the bottom saying what exactly the method returns.

Reply:
Added a few lines.